tests/integration/acl.editposts.feature

```diff
@@ -67,7 +67,6 @@ Feature: Access Control when modifying posts
     @rolesEnabled
     Scenario: Anonymous users can not edit posts
         Given that I want to update a "Post"
```

Comment:
Yea thats probably right. This is a bit rushed for Uchaguzi but hopefully we can clean it up next week and bring it back to core. There will definitely be some confusion between 401 and 403 responses